### ISSUE TITLE
MSCLoader_Core\core.unity3d removed upon installing

### ISSUE THREAD

**Issue by headcase0570** [piotrulos/MSCModLoader#120]
#### Info
* **OS version/build:** Windows 10 Pro version 20H2 19042.746
* **Architecture:** x64
* **MSCLoader version:** MSCLoader-147-1-1-8-1607548959
* **Type of issue:** Bug

#### Description
When running the 'Install MSCloader' action, I get an error about a missing file. (\Mods\Assets\MSCLoader_Core\core.unity3d) After re-installing all mod files and manually confirming the file's presence, I run Install MSCloader again, and again it shows the file as missing. 
In short, using the Install MSCloader causes an essential file to be removed, which renders it unable to continue.

![screenshot 86](https://user-images.githubusercontent.com/66142620/105635496-5a054480-5e63-11eb-8c60-61e707a98fef.png)
![screenshot 87](https://user-images.githubusercontent.com/66142620/105635497-5a9ddb00-5e63-11eb-8159-9f8ce79245e5.png)

Logfile
[mscloader log.txt](https://github.com/piotrulos/MSCModLoader/files/5862565/mscloader.log.txt)

**Comment by headcase0570** [comment]
I fixed it by removing all old MSCloader files and doing a 'fresh' installation.
